SUMMARY.md

```diff
@@ -50,7 +50,8 @@
     * [Shallow Tests](./docs/404.md)
     * [Narrow Tests](./docs/404.md)
     * [UI Testing](./docs/404.md)
-      * [Selenium WebDriver](./docs/404.md)
+      * [Selenium WebDriver](/docs/selenium-webdriver.md)
+	  * [Watir WebDriver](/docs/watir-webdriver.md)
```

Comment:
Q: is Watir article meant to be a sub-bullet-point of _Selenium WebDriver_ or of _UI Testing_?